actors/controllers.py

```diff
@@ -92,6 +92,10 @@ def get(self):
                     else:
                         logger.debug("Actor's cron is not activated, but next execution will be incremented")
                         actors_store[actor_id, 'cron_next_ex'] = Actor.set_next_ex(actor, actor_id)
+                elif self.cron_execution_datetime(actor) == 2:
```

Comment:
Putting the next ex and logger here instead of "cron_execution_datetime" will work, but it means that the execution won't be started immediately. It might be wise to move this stuff to "cron_execution_datetime" and have a simple, "return cron_exection <= now" instead. I do like the logger.debug though. So maybe add an if in that function before the return just to account for that.